packages/sdks-tests/src/snippet-tests/advanced-child.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from '@playwright/test';
+import { findTextInPage, test, verifyBlocks } from '../helpers/index.js';
+
+test.describe('Adding advanced child blocks in custom components', () => {
+  test('render two divs with the attribute builder-path', async ({ page, packageName }) => {
+    test.skip(!['react'].includes(packageName));
+
+    await page.goto('/advanced-child');
+
+    await page.waitForSelector('div[builder-path]', { timeout: 5000 });
+
+    const div = page.locator('div[builder-path]');
```

Comment:
- we should avoid custom `timeout` values in playwright selectors
- we should avoid `waitForSelector` in general: https://playwright.dev/docs/api/class-elementhandle#element-handle-wait-for-selector
- you should be able to combine these into:

```suggestion
    const div = page.locator('div[builder-path]').waitFor();
```